src/main/java/com/redislabs/modules/rejson/JReJSON.java

```diff
@@ -121,23 +125,14 @@ public JReJSON(Pool<Jedis> jedis) {
         this.client = jedis;
     }
 
-    /**
-     *  Helper to check for errors and throw them as an exception
-     * @param str the reply string to "analyze"
-     * @throws RuntimeException
-     */
-    private static void assertReplyNotError(final String str) {
-        if (str.startsWith("-ERR"))
-            throw new RuntimeException(str.substring(5));
-    }
-
     /**
      * Helper to check for an OK reply
      * @param str the reply string to "scrutinize"
      */
     private static void assertReplyOK(final String str) {
-        if (!str.equals("OK"))
-            throw new RuntimeException(str);
+        if (str == null) {
```

Comment:
@sazzad16 are you sure the only two options are "OK" or `null`?

Reply:
@gkorland Yes.

Reply:
@gkorland Currently this method never throws RE. It only generates NPE when `str` is null.